web/src/drafts_overlay_ui.ts

```diff
@@ -225,6 +227,21 @@ export function launch(): void {
             update_bulk_delete_ui();
         });
 
+        new ClipboardJS("#drafts_table .overlay_message_controls .copy-overlay-message", {
+            text(trigger): string {
+                const draft_id = $(trigger).attr("data-draft-id")!;
+                const draft = drafts.draft_model.getDraft(draft_id);
+                if (!draft) {
+                    return "";
+                }
+                return draft?.content ?? "";
```

Comment:
You don't have to use the conditional check if you are already using the optional chaining operator with `drafts` and vice versa.